src/test/docker/docker-dropwizard-example/dockerBuildImage.bat

```diff
@@ -0,0 +1 @@
+docker build -t example/dropwizard:latest .
```

Comment:
Why are helper bat/sh necessary? These should just be in the docs (as it looks like they are).